website/docs/docs/bi-directional-contract-testing/provider.md

```diff
@@ -5,11 +5,11 @@ sidebar_label: Provider
 
 ## Principles
 
-- Testing the provider API is your responsibility, PactFlow simply ensures the specification is compatible with any consumers
-- Garbage in, Garbage out - PactFlow trusts any provider contract that is provided. This is true, whether or not it has actually been tested.
-- When using the BYO functional API testing strategy, you must take responsibility for ensuring your API is compatible with (and ideally, implements) any specification.
-- Code based approaches are generally preferred because they are less likely to drift from implementation. For example, using tools that generate OAS definitions from code/types are more likely to be reliable
-- When supported, test based approaches such as ReadyAPI Functional test suites/postman collections, may also be more reliable, as they have embedded testing information in them. PactFlow will be able to only compare responses that have been tested and therefore are reliable.
+- Testing the provider API is your responsibility. PactFlow simply ensures the specification is compatible with any consumers
+- Garbage in, Garbage out - PactFlow trusts any provider contract provided. This is true, whether it has been tested or not.
+- When using the BYO functional API testing strategy, you must ensure your API is compatible with (and ideally, implements) any specification.
+- Code-based approaches are generally preferred because they are less likely to drift from implementation. For example, using tools that generate OAS definitions from code/types is more reliable
+- When supported, test-based approaches such as ReadyAPI Functional test suites/postman collections, may also be more reliable, as they have embedded testing information in them. PactFlow only compare responses that have been tested and therefore are reliable.
```

Comment:
The rephrasing of this implies that we can parse test results, which we can't. Suggestion:

```suggestion
- When supported, test-based approaches such as ReadyAPI Functional test suites/postman collections, may also be more reliable, as they have embedded testing information in them. Uploading only the tested parts of the provider contract to PactFlow improves the guarantees we can provide.
```